invenio_accounts/tasks.py

```diff
@@ -28,9 +28,12 @@
 
 from celery import shared_task
 from flask import current_app
+from flask_mail import Message
 
 
 @shared_task
-def send_security_email(msg):
+def send_security_email(data):
     """Celery task to send security email."""
+    msg = Message()
+    msg.__dict__.update(data)
```

Comment:
@lnielsen @eamonnmag please check following solution to avoid security issues with pickle serialization. Thanks